### PR TITLE
Enable ts.

### PR DESCRIPTION
**Pull request by maxime-rainville**

### CODE REVIEW

File: client/src/containers/ThumbnailView/ThumbnailView.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import i18n from 'i18n';
 import React, { Component } from 'react';
 import { inject } from 'lib/Injector';
```
Comment (maxime-rainville) [inline client/src/containers/ThumbnailView/ThumbnailView.tsx]: This is what we are trying to fix

Comment (ScopeyNZ) [reply client/src/containers/ThumbnailView/ThumbnailView.tsx]: Can you give more detail? Does it not like the weird webpack globals? I've been thinking about doing some fairly "simple" work to remove this annoying global and move injector to `@silverstripe/react-injector` - the import would become a valid `import { inject } from '@silverstripe/react-injector';`.

Comment (maxime-rainville) [reply client/src/containers/ThumbnailView/ThumbnailView.tsx]: Sorry ... that was specifically meant for @dnsl48 who had context on the issue.

Comment (maxime-rainville) [reply client/src/containers/ThumbnailView/ThumbnailView.tsx]: Basically, TS spit the dummy when you're trying to import a dependency expose from admin with the `expose-loader`.

That specific line throws that error.

**Comment by maxime-rainville** [comment]
We had discussion about this internally and decided it wasn't achievable in the current release line.

**Pull request closed without merging**